charts/wga/values.yaml

```diff
@@ -7,10 +7,23 @@
 ## @param global.imageRegistry Global Docker image registry
 global:
   imageRegistry: "ghcr.io/kraudcloud"
+  privateImageRegistry: "ctr.0x.pt"
 
 ## @section Common parameters
 ##
 
+welcome:
+  image: 
+    name: "wga-welcome"
+    tag: "main"
+    pullPolicy: ""
+
+  service:
+    type: "ClusterIP"
+
+  annotations: {}
+  labels: {}
+
```

Comment:
```suggestion
```

These are unused and unlikely to be useful, since you're integrating welcome into an existing pod, and only pods have annotations or labels